src/test/java/ay2021s1_cs2103_w16_3/finesse/logic/commands/ListExpenseCommandTest.java

```diff
@@ -30,30 +29,11 @@ public void setUp() {
     }
 
     @Test
-    public void execute_showsExpenseOnly() {
-        expectedModel.updateFilteredTransactionList(PREDICATE_SHOW_ALL_EXPENSES);
+    public void execute_showsAllExpensesOnly() {
```

Comment:
```suggestion
    public void execute_showsAllExpenses() {
```

Sounds weird otherwise. :/